### PR TITLE
Code cleanup in Cmsutil.crypto

### PR DESCRIPTION
**Pull request by ckelleyRH**

### CODE REVIEW

File: base/util/src/main/java/com/netscape/cmsutil/crypto/CryptoUtil.java [hunk 1]
```diff
@@ -193,25 +193,25 @@
...
+            //        SSLSocket.TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256,
+            SSLSocket.TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256,
+
+            SSLSocket.TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA384,
+            SSLSocket.TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,
+            /*
+                    SSLSocket.TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384,
+                    SSLSocket.TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,
+                    SSLSocket.TLS_DHE_RSA_WITH_CHACHA20_POLY1305_SHA256
+            */
```
Comment (edewata) [inline base/util/src/main/java/com/netscape/cmsutil/crypto/CryptoUtil.java:214]: I think for something like this we want to use single-line comments:
```
static public final Integer[] clientECCiphers = {
        // SSLSocket.TLS_ECDH_RSA_WITH_AES_128_CBC_SHA,
        // SSLSocket.TLS_ECDH_RSA_WITH_AES_256_CBC_SHA,
        SSLSocket.TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA,
        ...
};
```

File: base/util/src/main/java/com/netscape/cmsutil/crypto/CryptoUtil.java [hunk 2]
```diff
@@ -1431,51 +1427,47 @@ public static String getSKIString(X509CertImpl cert) throws IOException {
...
-                new OBJECT_IDENTIFIER(csOID.toString());
-/*
+        OBJECT_IDENTIFIER jssOID = new OBJECT_IDENTIFIER(csOID.toString());
+        /*
         return getExtensionFromCertTemplate(certTemplate, jssOID);
-    }
-    public static org.mozilla.jss.netscape.security.x509.Extension getExtensionFromCertTemplate(CertTemplate certTemplate, org.mozilla.jss.asn1.OBJECT_IDENTIFIER jssOID) {
-*/
+            }
+            public static org.mozilla.jss.netscape.security.x509.Extension getExtensionFromCertTemplate(CertTemplate certTemplate, org.mozilla.jss.asn1.OBJECT_IDENTIFIER jssOID) {
```
Comment (edewata) [inline base/util/src/main/java/com/netscape/cmsutil/crypto/CryptoUtil.java:1437]: I think the indentation will make it more confusing. Should we just remove the whole comment?

**Review by edewata — APPROVED** [review]
I have some minor comments, but it looks good otherwise.

File: base/util/src/main/java/com/netscape/cmsutil/crypto/CryptoUtil.java (hunk 1 above)
Comment (ckelleyRH) [reply base/util/src/main/java/com/netscape/cmsutil/crypto/CryptoUtil.java:214]: Ack

File: base/util/src/main/java/com/netscape/cmsutil/crypto/CryptoUtil.java (hunk 2 above)
Comment (ckelleyRH) [reply base/util/src/main/java/com/netscape/cmsutil/crypto/CryptoUtil.java:1437]: Yes, I agree.

**Pull request merged**